plugin.video.dailymotion_com/default.py

```diff
@@ -197,45 +217,45 @@ def listVideos(url):
     if content['has_more']:
         currentPage = content['page']
         nextPage = currentPage+1
-        addDir(translation(30001)+" ("+str(nextPage)+")", url.replace("page="+str(currentPage), "page="+str(nextPage)), 'listVideos', "")
-    if forceViewModeNew:
-        xbmc.executebuiltin('Container.SetViewMode('+viewModeNew+')')
+        addDir(translation(30001)+" ("+str(nextPage)+")", url.replace("page="+str(currentPage), "page="+str(nextPage)), 'listVideos', _ipath + "next_page2.png")
+    xbmcplugin.setContent(pluginhandle, "episodes")
+    xbmc.executebuiltin('Container.SetViewMode(54)')
```

Comment:
This isn't allowed.

Reply:
Can you clarify, please
The addon had this this line before
`xbmc.executebuiltin('Container.SetViewMode('+viewModeNew+')')`
and I replaced it with 
`xbmc.executebuiltin('Container.SetViewMode(54)')`

Is SetViewMode not allowed anymore? it does work correctly as intended though.

Reply:
Corrected it to use only if user setup the relevant modes in settings